### PR TITLE
tools: Port to Python 3

### PR DESCRIPTION
**Pull request by martinpitt**
Keep tap-{gtester,driver} as python 2, as they are also run on
RHEL/CentOS images. But they also work with Python 3 now.

 * [x] image-refresh ubuntu-1604

### CODE REVIEW

**Comment by martinpitt** [comment]
This needs a new ubuntu-1604 image to get python3 into its pbuilder. The other images already have it.

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-zgqmz.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9500-20180626-103232/

**Comment by cockpituous** [comment]
image-refresh ubuntu-1604 done: https://github.com/cockpituous/cockpit/commits/image-refresh-ubuntu-1604-20180626-110303

**Comment by martinpitt** [comment]
@Gundersanne : I assigned this to you for review, to get you into the mood and show you around more places of Cockpit. This is a rather easy one which self-tests. For any review, please don't shy away to ask about anything that's unclear - often these are a sign that some change needs a better commit message, or a comment, or needs more legible code, and so on.

File: tools/min-base-version [hunk 1]
```diff
@@ -27,7 +27,7 @@ from glob import glob
 
 proj_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 
-max_version = 0
+max_version = ''
```
Comment (Gundersanne) [inline tools/min-base-version]: A bit further on there's a `max_version == 0` check. Won't that always be False now?

Comment (martinpitt) [reply tools/min-base-version]: Well spotted, thanks! Fixed.

**Pull request merged**